src/main/resources/openapi/token-1.0.yaml

```diff
@@ -0,0 +1,113 @@
+openapi: 3.0.0
+info:
+  title: mod-authtoken API
+  version: v1
+paths:
+  # NOTE This module relies on the its folio-vertx-lib dependency to handle the /_/tenant
+  # route. Please see its yaml specification for /_/tenant here:
+  # https://github.com/folio-org/folio-vertx-lib/blob/master/core/src/main/resources/openapi/tenant-2.0.yaml
+  /token:
```

Comment:
@julianladisch  see what you think. I'm thinking this comment is probably enough to address the handing of `/_/tenant`.

Reply:
I agree.
The official documentation link is https://dev.folio.org/reference/api/#folio-vertx-lib

Reply:
Ok great, thanks Julian. And thanks for pointing out where the generated docs end up. I'll use that ink instead.